cmd/template/advanced/files/workflow/github/github_action_goreleaser.yml.tmpl

```diff
@@ -16,14 +16,21 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: '1.21.1'
-      -
-        name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v5.0.0
+          go-version: '1.22.x'
+        {{if or ( .AdvancedOptions.htmx ) ( .AdvancedOptions.tailwind )}}
```

Comment:
Indent all template statements to the beginning of the line. In YAML, curent indentation causes unwanted spaces to be inserted